ACMILabs/playlist-label
# Update/1436 deploy on master only

Resolves ACMILabs/xos#1436

- Made codefresh deploy to balena only on master pushes

### Acceptance Criteria
- [X] Deploy to balena only on master pushes

### Relevant design files
* None

### Testing instructions
1. Check that the push on this branch did not deploy https://g.codefresh.io/build/5e672e01bdf5efd15647e949. Alternatively, make an empty commit and push to this branch `git commit --allow-empty -m "#1436 Trigger codefresh"` and check that it doesn't deploy
2. On merge we can check that it deploys on master

### DoD
For requester to complete:
~New logic has been documented~
~New logic has appropriate unit tests~
~Changelog has been updated if necessary~
~Deployment / migration instruction have been updated if required~
